spec/controllers/sign_up/completions_controller_spec.rb

```diff
@@ -346,13 +355,32 @@
       end
 
       it 'logs a reproofing event upon reproofing' do
-        user.profiles.create(verified_at: Time.zone.now, active: true, activated_at: Time.zone.now)
+        unverified_profile = user.profiles.first
+        verified_profile = create(:profile, :verified, user: user)
+
         stub_sign_in(user)
         subject.session[:sp] = {
           ial2: false,
           issuer: 'foo',
           request_url: 'http://example.com',
         }
+
+        expect(unverified_profile.activated_at).to be_present
+        expect(unverified_profile.active).to eq false
+        expect(unverified_profile.deactivation_reason).to be_nil
+        expect(unverified_profile.fraud_review_pending?).to eq(false)
+        expect(unverified_profile.gpo_verification_pending_at).to be_nil
+        expect(unverified_profile.initiating_service_provider).to be_nil
+        expect(unverified_profile.verified_at).to be_nil
+
+        expect(verified_profile.activated_at).to be_present
+        expect(verified_profile.active).to eq false
+        expect(verified_profile.deactivation_reason).to be_nil
+        expect(verified_profile.fraud_review_pending?).to eq(false)
+        expect(verified_profile.gpo_verification_pending_at).to be_nil
+        expect(verified_profile.initiating_service_provider).to be_nil
+        expect(verified_profile.verified_at).to be_present
+
         expect(@irs_attempts_api_tracker).to receive(:idv_reproof)
```

Comment:
@soniaconnolly: @theabrad and I think that `Profile#has_proofed_before?` may need to be refactored to rely on `activated_at` AND `verified_at`. It currently considers `activated_at` only.

@benjaminchait this is IRS-related code, should we loop in anyone else?